external-libraries.md

```diff
@@ -133,6 +133,20 @@ URL:     https://github.com/weisJ/jsvg
 License: MIT
 ```
 
+```yaml
+Id:      com.googlecode.javaewah:JavaEWAH
+Project: JavaEWAH
+URL:     https://github.com/lemire/javaewah
+License: Apache-2.0
+```
+
+```yaml
+Id:      com.googlecode.plist:dd-plist
+Project: com.dd.plist - A Java library for working with property lists
+URL:     https://github.com/3breadt/dd-plist
+License: MIT
+```
+
```

Comment:
Do not edit external-libraries by hand. It will be edited when releasing a new version. There are too many possible errors. Please revert the changes to this file.

Reply:
Na actually it looks good so far

Reply:
The concrete change is OK for me. However, the changes below are not.

`commons-collections:commons-collections:3.2.2` is added, but no added entry in `external-libraries.md`. Thus, either all or none.

Reply:
So lets keep just the license and revert the other changes. Have to be done anyway on next release. That what I call in german "verschwendete Liebesmüh".